.github/workflows/any-pr.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        node-version: [16.x, 18.x]
```

Comment:
👍 Since this repo is public shall we continue to support node 16?

I think it's fair enough that we drop support for it just something to check.

Not sure if your `neon` change will be backwards compatible with `16` anyway?

Reply:
Nope, it only works with node 18 and above, it uses n-api I think

Reply:
Cool lets just double check we are cool with this so people are aware 😄

Reply:
I will add on the release PR.

Reply:
Sounds good. Mark with a BREAKING CHANGE, and people can continue to use the previous versions